pages/understanding-json-schema/reference/non_json_data.md

```diff
@@ -7,84 +7,128 @@ section: docs
 
 <Star label="New in draft 7" />
 
-JSON schema has a set of [keywords](../../learn/glossary#keyword) to describe and optionally validate
-non-JSON data stored inside JSON strings. Since it would be difficult to
-write validators for many media types, JSON schema validators are not
-required to validate the contents of JSON strings based on these
-keywords. However, these keywords are still useful for an application
-that consumes validated JSON.
+JSON schema has a set of [keywords](../../learn/glossary#keyword) to describe and optionally validate non-JSON data stored inside JSON strings that are stored or transmitted. Due to the difficulty in writing validators for all media types, JSON schema validators are not required to validate the contents of JSON strings based on these keywords. However, these keywords are useful for applications that consume validated JSON during the processes of encoding and decoding data.  
 
 <Keywords label="single: contentMediaType single: media; contentMediaType" />
 
-## contentMediaType
+## contentMediaType and contentEncoding
 
-The `contentMediaType` keyword specifies the MIME type of the contents
+The `contentMediaType` keyword specifies the MIME type of the content
 of a string, as described in [RFC 2046](https://tools.ietf.org/html/rfc2046).
-There is a list of [MIME types officially registered by the IANA](http://www.iana.org/assignments/media-types/media-types.xhtml),
-but the set of types supported will be application and operating system dependent.
-Mozilla Developer Network also maintains a [shorter list of MIME types that are important for the web](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types)
+The IANA has oficially registered [a comprehensive list of MIME types](http://www.iana.org/assignments/media-types/media-types.xhtml),
+but the set of supported types depends on the application and operating system.
+Mozilla Developer Network maintains a [shorter list of MIME types that are important for the web](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types)
 
-<Keywords label="single: contentEncoding single: media; contentEncoding" />
+Example: 
```

Comment:
I think this should be a heading or at least in bold or something.

```suggestion
### Example
```

Reply:
Agreed and adopted the bold suggestion